packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js

```diff
@@ -441,75 +344,48 @@ async function addIntent(context, botName, resourceName, serviceMetadata, intent
   };
 }
 
-async function addUtterance(context, intentName, botName, resourceName, serviceMetadata) {
-  const { inputs } = serviceMetadata;
-  const { amplify } = context;
-  const addAnotherUtteranceQuestion = {
-    type: inputs[24].type,
-    name: inputs[24].key,
-    message: inputs[24].question,
-    default: inputs[24].default,
-  };
-  const utteranceQuestion = {
-    type: inputs[13].type,
-    name: inputs[13].key,
-    message: inputs[13].question,
-    validate: amplify.inputValidation(inputs[13]),
-  };
-  let addAnotherUtterance = true;
+async function askIntent(resourceName) {
+  return prompter.input(
+    'Give a unique name for the new intent:',
+    {
+      initial: resourceName,
+      validate: matchRegex(
+        /^([A-Za-z]_?){1,100}$/,
+        'Intent name can only contain letters, cannot be empty, and must be no longer than 100 characters'
```

Comment:
This regex looks like it can contain letters and underscore?

Reply:
updated the message